_sass/_partners.scss

```diff
@@ -0,0 +1,98 @@
+@media screen and (min-width: 280px) {
+    .each-partner {
+        width: 100%;
+        background-color: #ffffff;
+    }
+    .partner-logo{
+      width: 200px;
+      height: 100px;
```

Comment:
Please remove the height attribute from this line, and set the width in the line above to `max-width: 100%;` instead. That will make it responsive within the container.